README.md

```diff
@@ -19,16 +19,16 @@ the [Greenlight](https://blockstream.com/lightning/greenlight/) infrastructure.
 
 ## Build
 
-### Build the lightning_tookit plugin
+### Build the lightning_toolkit plugin
```

Comment:
What is the `lightning_toolkit` plugin?

Reply:
It's from 3 years ago.. possibly the early codename for our native sdk.

I was only fixing the typo but I'll update it to "Breez SDK - Native (Greenlight Implementation)" 😄

Reply:
@dangeross I've updated our README to be more up-to-date & consistent with the structure of Misty Breez.

[chore: update README](https://github.com/breez/c-breez/pull/919/commits/0cd16ba327035b171c3f1fbdc57d4ed1fdef105d)